pdr_backend/lake/gql_data_factory.py

```diff
@@ -130,17 +205,16 @@ def _update(self):
             fin_ut -- a timestamp, in ms, in UTC
         """
 
-        for _, table in self.record_config["tables"].items():
-            filename = table._parquet_filename()
-            st_ut = table._calc_start_ut(filename)
+        for table_name, table in self.record_config["tables"].items():
+            st_ut = table.csv_data_store.get_first_timestamp(table_name)
```

Comment:
On the following PR, I changed the logic which is at this part. We have to decide which one that we have to use

https://github.com/oceanprotocol/pdr-backend/pull/751/files#